src/main/resources/assets/securitycraft/lang/pt_br.json

```diff
@@ -0,0 +1,1198 @@
+	"help.securitycraft.briefcase.info": "A pasta é uma unidade de armazenamento portátil com travamento teclado contendo 12 slots.Depois de elaborá-lo, clique com a direita para configurar a senha de 4 dígitos para usar a partir de então.Clique com o direito de novo e insira o código para interagir com os itens.Para remover o código de uma pasta ou alterar seu proprietário, coloque -o em seu clique sem direito e segura um trocador de chave universal/trocador de proprietários universais em sua mão -de -obra.",
+	"help.securitycraft.universal_key_changer.info": "O trocador de chave universal permite alterar a senha de blocos protegidos pela senha que você possui.Clique com a direita um bloco protegido por senha para abrir uma GUI e digite a nova senha que desejar.",
+	"help.securitycraft.sentry.info": "O Sentry possui três modos que podem ser alternados clicando na direita.Ocle (nunca atacará), camuflagem (apenas atacará jogadores inimigos em seu alcance) e agressivo (atacará mobs hostis e jogadores inimigos).Clique com o direito com um módulo de disfarce contendo um bloco completo e opaco (por exemplo, pedra, mas não vidro ou escada) fará com que o próprio disfarce.O módulo pode ser alterado diretamente ou extraído clicando com um modificador de bloco universal.Para remover a sentinela, clique com o Sneak-Right It.",
+	"help.securitycraft.crystal_quartz_item.info": "Crystal Quartz é usado para criar blocos de quartzo de cristal.",
+	"help.securitycraft.speed_module.info": "O módulo de velocidade permite que alguns dos blocos da SecurityCraft operem mais rápido do que o normal.",
+	"help.securitycraft.portable_tune_player.info": "O jogador de música portátil pode tocar uma música que foi salva anteriormente.Para economizar uma música, clique com a direita um sistema de segurança sonora que já registrou uma música.Para tocar a música, clique com a direita o jogador de música portátil.Para cancelar a reprodução, basta clique com o player de música portátil novamente ou parar de segurá -lo.",
+
+	"gui.securitycraft:scManual.disabled": "Desabilitada",
+	"gui.securitycraft:scManual.recipe.keycard_lv1": "Desabilitada",
+	"gui.securitycraft:scManual.recipe.keycard_lv2": "Desabilitada",
+	"gui.securitycraft:scManual.recipe.keycard_lv3": "Desabilitada",
+	"gui.securitycraft:scManual.recipe.keycard_lv4": "Desabilitada",
+	"gui.securitycraft:scManual.recipe.keycard_lv5": "Desabilitada",
+	"gui.securitycraft:scManual.recipe.limited_use_keycard": "Desabilitada",
+	"gui.securitycraft:scManual.intro.1": "Começando com SecurityCraft",
+	"gui.securitycraft:scManual.intro.2": "por GeForce, BL4CKSCOR3, Redstone_Dubstep e ChainMailpickExe",

```

Comment:
Just to correct some of our names, that phrase would be "por Geforce, bl4ckscor3, Redstone_Dubstep e ChainmailPickaxe."

Reply:
You can also translate and add the following line blow this one with your preferred name:
`"gui.securitycraft:scManual.author": "Translated by FITFC",`